template/src/ontology/Makefile.jinja2

```diff
@@ -1243,7 +1246,7 @@ public_release:
 	$(GITHUB_RELEASE_PYTHON) --release $(TAGNAME) $(RELEASEFILES)
 {% else  %}
 
-RELEASE_ASSETS_AFTER_RELEASE=$(foreach n,$(RELEASE_ASSETS), ../../$(n)){% if project.sssom_mappingset_group is defined %}{% if project.sssom_mappingset_group.release_mappings %} $(foreach n,$(MAPPING_FILES), ../$(n)){% endif %}{% endif %}
+RELEASE_ASSETS_AFTER_RELEASE=$(foreach n,$(RELEASE_ASSETS), ../../$(n)) $(foreach n,$(RELEASED_MAPPING_FILES), ../$(n))
```

Comment:
Check the definition of `RELEASED_MAPPING_FILES`:

```Makefile
RELEASED_MAPPING_FILES = $(foreach p, $(RELEASED_MAPPINGS), $(MAPPINGDIR)/$(p).sssom.tsv)
```

(This is identical to the definition of `MAPPING_FILES`, except that the source list is `RELEASED_MAPPINGS` rather than `MAPPINGS`.)

Assuming, for example, a list of mapping sets to be released that contains `seta setb`, we will have

```Makefile
RELEASED_MAPPING_FILES = ../mappings/seta.sssom.tsv ../mappings/setb.sssom.tsv
```

This is the paths to the mapping set files, relative to the `src/ontology` directory, so notice how they already include the `../` needed to go back to `src`.

So all we need to do to get the list of files after they have been put to the release (top-level) directory is to prepend _one_ another `../`, which we are doing here.

Yes, this is hardcoding the fact that `RELEASEDIR` is the top-level directory. I didn’t see any problem with that since this was _already_ the behaviour of the definition of `RELEASE_ASSETS_AFTER_RELEASE` for all the other assets.

Reply:
Of course, duh! Thanks for induliging.